scripts/zkai_attention_kv_sixteen_head_air_private_softmax_table_lookup_gate.py

```
@@ -0,0 +1,1091 @@
+        try:
+            validate_payload(mutated, allow_missing_mutation_summary=True)
+        except AttentionKvAirPrivateSoftmaxTableLookupGateError as err:
+            cases.append({"name": name, "rejected": True, "error": str(err)})
+        else:
+            cases.append({"name": name, "rejected": False, "error": "accepted mutation"})
+    return cases
+
+
+def validate_mutation_spec() -> None:
+    if len(EXPECTED_MUTATION_NAMES) != EXPECTED_MUTATION_COUNT:
+        raise AttentionKvAirPrivateSoftmaxTableLookupGateError("mutation spec count drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)